### DISCUSSION TITLE
4.7.0

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#709] [Announcements]
⚠ 由于X64架构编译选项调整，可能导致更新失败，如果更新失败需要先卸载旧版应用，重启系统后再查看安装教程使用**脚本方式**安装此版本应用
如需备份恢复设置，可使用设置页中开发者选项中的导出与导入设置功能
⚠ 无论Win10/Win11，暂时都只支持脚本方式安装，安装成功后续版本才可以直接使用msix方式安装
***
#### ADDED
- 视频详情页播放列表功能补全：滚动到顶部、滚动到底部、滚动到当前视频 #678
- 支持发送带图片评论，自己发送的评论点右键可删除 #689 
- 支持设置推荐页、搜索页、动态页内容过滤 #698 
#### CHANGED
- 调整视频详情页收藏菜单样式、支持鼠标中键侧键和Ctrl+S快捷关闭保存收藏选项 #682
- 设置页设置项改用WinUI设置控件  #705 
#### FIXED
- 修复动态中用户链接打开错误 #686 
- 修复视频点不喜欢报错 #687 
- 修复快捷键设置倍速报错 #694 
- 关闭X64架构的nativeToolchain编译选项，尝试修复二维码登录时打开应用就闪退问题 #702 

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.0'>4.7.0</a>.</em>

**Comment by Danmo04** [comment]
卸载旧版重启后仍然安装失败

**Reply by ywmoyue** [reply]
> 卸载旧版重启后仍然安装失败

具体报什么错误

**Reply by ywmoyue** [reply]
方便截图看下命令执行后报什么错误吗

**Reply by Danmo04** [reply]
我使用的是双击msix的安装方法

**Reply by ywmoyue** [reply]
> 我使用的是双击msix的安装方法

请使用脚本方式运行Install.ps1安装

**Reply by Danmo04** [reply]
已成功，感谢

**Comment by SEKAI-NO-YAMADA** [comment]
您好，卸载重启安装后还是无法安装，win11
<img width="493" alt="屏幕截图 2024-07-20 201013" src="https://github.com/user-attachments/assets/82bc98da-78dc-4048-a665-ed30f4df5d29">

**Reply by ywmoyue** [reply]
请使用脚本方式安装

**Reply by SEKAI-NO-YAMADA** [reply]
已解决，感谢
